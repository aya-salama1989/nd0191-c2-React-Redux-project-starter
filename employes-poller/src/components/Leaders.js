
const Leaders = ()=>{

    return(<li>What </li>);
}

export default Leaders;