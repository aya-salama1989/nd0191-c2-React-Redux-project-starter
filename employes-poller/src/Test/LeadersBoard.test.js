import { render } from "@testing-library/react";
import * as React from "react";
import Leaders from "../components/Leaders";
import { Provider } from "react-redux";
import { store } from '../index'


describe("Leaders", () => {
  it("will catch a snapshot for leaders", () => {
    const view = render(
        <Provider store={store}>
          <Leaders />
        </Provider>
    );
    expect(view).toMatchSnapshot();
  });
});
