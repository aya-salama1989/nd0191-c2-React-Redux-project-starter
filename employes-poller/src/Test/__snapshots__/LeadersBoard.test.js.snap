// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Leaders will catch a snapshot for leaders 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <ul>
        <div
          class="mainDiv"
        >
          <div
            class="third"
          >
            <img
              alt="Avatar of user"
              class="avatar"
              src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
            />
          </div>
          <div
            class="twoThird"
          >
            <h4>
              Name: Sarah Edo
            </h4>
            <span
              class="userDataItem"
            >
              Ranking: 1
            </span>
            <hr />
            <span
              class="userDataItem"
            >
              Asked Questions: 2
            </span>
            <span
              class="userDataItem"
            >
              Answered Questions: 4
            </span>
          </div>
        </div>
        <div
          class="mainDiv"
        >
          <div
            class="third"
          >
            <img
              alt="Avatar of user"
              class="avatar"
              src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
            />
          </div>
          <div
            class="twoThird"
          >
            <h4>
              Name: Mike Tsamis
            </h4>
            <span
              class="userDataItem"
            >
              Ranking: 2
            </span>
            <hr />
            <span
              class="userDataItem"
            >
              Asked Questions: 2
            </span>
            <span
              class="userDataItem"
            >
              Answered Questions: 3
            </span>
          </div>
        </div>
        <div
          class="mainDiv"
        >
          <div
            class="third"
          >
            <img
              alt="Avatar of user"
              class="avatar"
              src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
            />
          </div>
          <div
            class="twoThird"
          >
            <h4>
              Name: Tyler McGinnis
            </h4>
            <span
              class="userDataItem"
            >
              Ranking: 3
            </span>
            <hr />
            <span
              class="userDataItem"
            >
              Asked Questions: 2
            </span>
            <span
              class="userDataItem"
            >
              Answered Questions: 2
            </span>
          </div>
        </div>
        <div
          class="mainDiv"
        >
          <div
            class="third"
          >
            <img
              alt="Avatar of user"
              class="avatar"
              src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
            />
          </div>
          <div
            class="twoThird"
          >
            <h4>
              Name: Zenobia Oshikanlu
            </h4>
            <span
              class="userDataItem"
            >
              Ranking: 4
            </span>
            <hr />
            <span
              class="userDataItem"
            >
              Asked Questions: 0
            </span>
            <span
              class="userDataItem"
            >
              Answered Questions: 1
            </span>
          </div>
        </div>
      </ul>
    </div>
  </body>,
  "container": <div>
    <ul>
      <div
        class="mainDiv"
      >
        <div
          class="third"
        >
          <img
            alt="Avatar of user"
            class="avatar"
            src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
          />
        </div>
        <div
          class="twoThird"
        >
          <h4>
            Name: Sarah Edo
          </h4>
          <span
            class="userDataItem"
          >
            Ranking: 1
          </span>
          <hr />
          <span
            class="userDataItem"
          >
            Asked Questions: 2
          </span>
          <span
            class="userDataItem"
          >
            Answered Questions: 4
          </span>
        </div>
      </div>
      <div
        class="mainDiv"
      >
        <div
          class="third"
        >
          <img
            alt="Avatar of user"
            class="avatar"
            src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
          />
        </div>
        <div
          class="twoThird"
        >
          <h4>
            Name: Mike Tsamis
          </h4>
          <span
            class="userDataItem"
          >
            Ranking: 2
          </span>
          <hr />
          <span
            class="userDataItem"
          >
            Asked Questions: 2
          </span>
          <span
            class="userDataItem"
          >
            Answered Questions: 3
          </span>
        </div>
      </div>
      <div
        class="mainDiv"
      >
        <div
          class="third"
        >
          <img
            alt="Avatar of user"
            class="avatar"
            src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
          />
        </div>
        <div
          class="twoThird"
        >
          <h4>
            Name: Tyler McGinnis
          </h4>
          <span
            class="userDataItem"
          >
            Ranking: 3
          </span>
          <hr />
          <span
            class="userDataItem"
          >
            Asked Questions: 2
          </span>
          <span
            class="userDataItem"
          >
            Answered Questions: 2
          </span>
        </div>
      </div>
      <div
        class="mainDiv"
      >
        <div
          class="third"
        >
          <img
            alt="Avatar of user"
            class="avatar"
            src="https://gravatar.com/avatar/984cd6ea2ab7a97be50364e928f88986?s=400&d=robohash&r=x"
          />
        </div>
        <div
          class="twoThird"
        >
          <h4>
            Name: Zenobia Oshikanlu
          </h4>
          <span
            class="userDataItem"
          >
            Ranking: 4
          </span>
          <hr />
          <span
            class="userDataItem"
          >
            Asked Questions: 0
          </span>
          <span
            class="userDataItem"
          >
            Answered Questions: 1
          </span>
        </div>
      </div>
    </ul>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
